mitigation is marked non-null but is null

Hi,

I am getting the following error by uploading a Dependency Check result:
```
Caught: com.fasterxml.jackson.databind.JsonMappingException: mitigation is marked non-null but is null
 at [Source: (StringReader); line: 1, column: 1622] (through reference chain: io.securecodebox.persistence.defectdojo.models.DefectDojoResponse["re
sults"]->java.util.ArrayList[0]->io.securecodebox.persistence.defectdojo.models.Finding["mitigation"])
com.fasterxml.jackson.databind.JsonMappingException: mitigation is marked non-null but is null
 at [Source: (StringReader); line: 1, column: 1622] (through reference chain: io.securecodebox.persistence.defectdojo.models.DefectDojoResponse["re
sults"]->java.util.ArrayList[0]->io.securecodebox.persistence.defectdojo.models.Finding["mitigation"])
        at com.fasterxml.jackson.databind.JsonMappingException.from(JsonMappingException.java:274)
        at com.fasterxml.jackson.databind.deser.SettableBeanProperty._throwAsIOE(SettableBeanProperty.java:623)
        at com.fasterxml.jackson.databind.deser.SettableBeanProperty._throwAsIOE(SettableBeanProperty.java:611)
        at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:143)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:324)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:187)
        at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer._deserializeFromArray(CollectionDeserializer.java:355)
        at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:244)
        at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:28)
        at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:324)
        at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:187)
        at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:322)
        at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4593)
        at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3548)
        at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3531)
        at io.securecodebox.persistence.defectdojo.service.FindingService.deserializeList(FindingService.java:48)
        at io.securecodebox.persistence.defectdojo.service.GenericDefectDojoService.internalSearch(GenericDefectDojoService.java:115)
        at io.securecodebox.persistence.defectdojo.service.GenericDefectDojoService.search(GenericDefectDojoService.java:124)
        at io.securecodebox.persistence.defectdojo.service.FindingService.getUnhandledFindingsForEngagement(FindingService.java:59)
        at io.securecodebox.persistence.defectdojo.service.FindingService$getUnhandledFindingsForEngagement.call(Unknown Source)
        at importToDefectDojo.call(importToDefectDojo.groovy:141)
        at importToDefectDojo$call.call(Unknown Source)
        at defectdojo.run(defectdojo.groovy:61)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
Caused by: java.lang.NullPointerException: mitigation is marked non-null but is null
        at io.securecodebox.persistence.defectdojo.models.Finding.setMitigation(Finding.java:30)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:141)
        ... 23 more
```
Should we set a default empty string for mitigation?

--- a/src/main/java/io/securecodebox/persistence/defectdojo/models/Finding.java
+++ b/src/main/java/io/securecodebox/persistence/defectdojo/models/Finding.java
@@ -58,7 +58,6 @@ public class Finding extends DefectDojoModel {
   Long test;
 
   @JsonProperty
-  @NonNull
   String mitigation;
 
   @JsonProperty

